onboarding/chat-moderators.md

```diff
@@ -13,36 +13,65 @@ Removal of the role is subject to the discretion of the Core team members and ma
 
 ## Chat Responsibilities and Guidelines
 
+### Take notes of any moderation action
+- The Discord server uses [Dyno bot](https://dyno.gg/bot) to aid in automation of certain moderation tasks,
+- When you moderate a user, you should add a note on the user in the `#chat-mod-actions` channel explaining the rule they broke and what action was taken (e.g. warned, threaded, etc).
+- You can include links to messages in another channel within user notes, if you feel the context of the message is relevant to the note.
+- You can check out some of the available [Dyno commands](https://dyno.gg/commands), but the ones you will most likely be using most often are `?notes` and `?note`.
+
 ### Maintain, follow, and enforce rules
 
 - As a moderator, you will be expected to **adhere to** and **uphold** the rules as detailed and maintained in the `#rules` channel in the Discord at all times. 
-- **`Delete` messages from all channels and threads.**
-    - Use discretion.
-    - Screenshot all messages to be deleted and post them in the `#chat-mod-actions` channel.
-    - Immediately delete content that breaks rules regarding hate speech, racism, and pornography. 
-    - Use discretion within reason regarding "bad etiquette".
-    - Malicious messages should be deleted.
-        - e.g. If somebody is attempting to get somebody to run `rm -rf /`.
+- **Change inappropriate nicknames**
+    - User nicknames must adhere to our rules and cannot be hateful, racist, sexual, etc. in nature, either overtly or vaguely.
+    - If possible, change the user's name to something more appropriate and let the user know of the name change in a private thread.
+    - Abuse of this permission will not be tolerated.
+- **Emphasize intent vs impact for disparaging comments**
+    - Users should be reminded that we are an inclusive community, and that their messages will be read by many different types of users.
+    - Users should also be reminded that even if their intent wasn't malicious, other users reading their message may be impacted negatively.
+        - An example of intent vs impact could be someone including a phrase that is hateful in nature towards a group of individuals, with the intent of just making a simple joke, but the impact of that group being targeted by a hateful comment.
```

Comment:
Looks pretty comparable to what I had in mind! The only thing I would consider including is adding "either implicitly or explicitly" to the end of the example. The reason is that I think this would be enough to include dog-whistle-type statements as well as statements that aren't very explicitly disparaging but would still potentially toe that line.

Reply:
Do you think that should go before the intent part (e.g. "...including a phrase that is either explicitly or implicitly hateful in nature towards a group of individuals...."), or after the impact part (e.g. "...but the impact of that group being explicitly or implicitly targeted by a hateful comment.")

I was leaning towards before intent, but it could probably fit in either place.

Reply:
Leaning towards before intent as well since that flows slightly better. Although it does ultimately get the point across either way!